includes/polling/os/f5.inc.php

```diff
@@ -14,7 +14,6 @@
         'sessions' => $sessions,
     );
 
-    $tags = compact('rrd_def');
```

Comment:
You accidentally deleted this line, please restore it.

Reply:
Replaced missing line